openrouteservice/src/main/java/org/heigit/ors/api/responses/routing/json/JSONLeg.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * This file is part of Openrouteservice.
+ *
+ * Openrouteservice is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU Lesser General Public License as published by the Free Software Foundation; either version 2.1
+ * of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this library;
+ * if not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.heigit.ors.api.responses.routing.json;
+
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import org.heigit.ors.routing.RouteLeg;
+import org.heigit.ors.routing.RoutePtStop;
+import org.heigit.ors.routing.RouteStep;
+import org.heigit.ors.util.PolylineEncoder;
+import org.locationtech.jts.geom.Coordinate;
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@ApiModel(value="JSONLeg", description = "Leg of a route")
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class JSONLeg {
+    @ApiModelProperty(value = "The type of the leg, possible values are currently 'walk' and 'pt'.", example = "pt")
```

Comment:
Shouldn't we better rename all `pt` instances to `public-transit` ?

Reply:
![image](https://user-images.githubusercontent.com/31958783/224936627-24b61ec9-a441-4192-bdc5-d181beb2a17b.png)
@koebi renamed `pt`  to `public-transport` and it seems okay to me...